test/json.js

```diff
@@ -6,6 +6,12 @@ var request = require('supertest')
 
 var bodyParser = require('..')
 
+/**
+ * @const
+ * whether current node version has brotli support
+ */
+var hasBrotliSupport = process.versions ? 'brotli' in process.versions : false
```

Comment:
It looks like the code coverage only shows one of these branches are ever run. Under what Node.js versions would the other branch run?